workflow-templates/appstore-build-publish.yml

```diff
@@ -68,24 +71,42 @@ jobs:
           php-version: ${{ env.PHP_VERSION }}
           coverage: none
 
+      - name: Install composer dependencies
+        # Continue if no composer.json
+        continue-on-error: true
```

Comment:
I think we should only continue if there is no composer.json data :/ but if there is and it fails to load it, e.g. because a lib is incompatible with php versions etc, we should not build without the vendor/ dir?

Reply:
That is fair!
Let's see if I can make that happen

Reply:
Done